doc/packet_trimming/packet-trimming-design.md

```diff
@@ -692,6 +771,15 @@ show
 |    |--- global [OPTIONS]
 |
 |--- mmu
+|
+|--- switch
+     |--- counters [OPTIONS]
+          |--- all [OPTIONS]
+          |--- trim [OPTIONS]
+          |--- detailed [OPTIONS]
+
+sonic-clear
+|--- switchcounters
```

Comment:
@nazariig In addition to 'show' commands, we also need gnmi support. Could you please add it in HLD.

Please use https://github.com/sonic-net/sonic-gnmi/pull/452 for reference.